efc/rpn_builder/parser/functions.py

```diff
@@ -681,6 +683,22 @@ def year_frac(dt1, dt2, tp=None):
     return v
 
 
+YMD_DATE_FORMAT = '%Y{0}%m{0}%d'
+DMY_DATE_FORMAT = '%d{0}%m{0}%Y'
+
+
+def date_func(key, cell_operand):
+    for date_format in (YMD_DATE_FORMAT, DMY_DATE_FORMAT):
+        for separator in '.-/':
+            try:
+                value = datetime.strptime(cell_operand.string, date_format.format(separator))
+                return getattr(value, key)
+            except:
```

Comment:
`except` without an exception class handle every exception, e.g. `KeyboardInterrupt` or `SystemExit`. You should specify the exception class.

Reply:
Handle exception may be slower than using `if` statement. Please, check it and use faster solution.